client/containers/forms/TechnologyFormContainer.js

```diff
@@ -0,0 +1,38 @@
+import {connect} from 'react-redux'
+import {TechnologyForm} from "../../components"
+import * as A from '../../actions'
+import * as COC from '../../components/componentConsts'
+import {NotificationManager} from 'react-notifications'
+import * as EC from "../../../server/errorcodes";
+import {SubmissionError} from "redux-form";
+
+const mapDispatchToProps = (dispatch, ownProps) => ({
+    onSubmit: (values) => {
+        console.log("You are in Technology initiate Form container ", values)
+        dispatch(A.addTechnologyOnServer(values)).then(json=>{
```

Comment:
Alway return dispatch on onSubmit, this will ensure that any SubmissionError you throw are added as error in redux-form and hence shown in form